backend/src/case_file_action/case_file_action.graphql

```diff
@@ -0,0 +1,16 @@
+type CaseFileAction {
+  actionId: String!
+  actionTypeCode: String!
+  actionCode: String!
+  actor: String!
+  date: Date!
+  displayOrder: Int
+  activeIndicator: Boolean
+  shortDescription: String!
+  longDescription: String!
+}
+
+type Query {
```

Comment:
This should not be exposed, the frontend shouldn't ever be requesting actions directly and in stead should be provided by the getCaseFile query instead